rocketpy/motors/fluid.py

```diff
@@ -62,7 +142,20 @@ def __str__(self):
         return f"Fluid: {self.name}"
 
     def to_dict(self, **kwargs):  # pylint: disable=unused-argument
-        return {"name": self.name, "density": self.density}
+        discretize = kwargs.get("discretize", False)
+        if not isinstance(self.density, (int, float, np.number)):
```

Comment:
this comparison could use the NUMERICAL_TYPES, no? 

Also, a comprehension here would be nice.